src/plays/smart-todo-list/styles.css

```diff
@@ -0,0 +1,334 @@
+:root {
+  --clr-primary: #0ce7f3;
+  --clr-secondary: #010326;
+  --clr-light: #fff;
+  --clr-active: #33ae10;
+  --clr-complete: #5454e8;
+  --clr-update: #00d866;
+  --clr-delete: #ff7849;
+}
+#wa_todo-list {
```

Comment:
It's always better to work on classes for styles than on ids.

Reply:
Fixed